diffpy/snmf/factorizers.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import scipy.optimize
+
+
+def lsqnonneg(coefficient, target):
+    """
```

Comment:
The first line of the docstring is what the code does.  Is it here and I am not seeing it or is it missing?

Reply:
I forgot to add it. I will do that now